### DISCUSSION TITLE
ゲームリプレイのバグ？

### DISCUSSION THREAD

**Discussion by KimamanaNeko** [smly/mjai.app#78] [Bug report] [answered]
![image](https://github.com/smly/mjai.app/assets/46703388/12437b69-9369-4ee8-b1bd-5e93ec8d4ac5)

https://mjai.app/games/16/13/2?k=2&s=24
確認お願いいたします

**Comment by smly** [comment, marked as answer]
バグレポートありがとうございます！とても助かります。暗槓のあとに予期せず 5m が手配に残っているようです。牌譜に問題はなく、可視化にバグがあるようです。今週を目処にバグ修正をします。

Thanks for the bug report! It is very helpful. There is an unexpected 5m left in hand after Ankan. This is a bug in the visualizer. I will fix it this week.

**Reply by smly** [reply]
遅くなりましたが修正しました！
Fixed. Thanks for reporting!
https://mjai.app/games/16/13/2?k=2&s=24
